src/components/internal-header/_internal-header.scss

```diff
@@ -46,6 +46,12 @@
   }
 }
 
+.hmrc-internal-header__logo-text {
+  display: inline-block;
```

Comment:
Have been looking at this locally and found an inconsitency between the margin between the text and logo. There is more margin on this version which I think is due to the element having `display: inline-block;` and whitespace (Take a look at https://developer.mozilla.org/en-US/docs/Web/API/Document_Object_Model/Whitespace#spaces_in_between_inline_and_inline-block_elements).
Thinking about this further, we probably want this element as an inline element as we don't want the logo and text to be split over multiple lines.
So, if you change this element to inline (either via the CSS or changing it to a span element and getting rid of the CSS rule) then we'll fix both issues.

Reply:
That makes sense, I'll change the CSS to keep it inline with HMRC banner for now. and thanks for the link :)